accounts/accounts/spiders/spider.py

```diff
@@ -10,32 +11,41 @@ class SpiderSpider(scrapy.Spider):
 
     def start_requests(self):
         headers = {
-            'Accept':'application/json, text/javascript, */*; q=0.01',
-            'Accept-Encoding': 'gzip,deflate,identity',
-            'User-Agent': 'Mozilla/5.0 (Linux; Android 4.2.1; en-us; Nexus 4 Build/JOP40D) AppleWebKit/535.19 (KHTML, like Gecko) Chrome/18.0.1025.166 Mobile Safari/535.19',
-            'Accept-Charset' : 'ISO-8859-1,utf-8;q=0.7,*;q=0.7',
-            'Accept-Language': 'en-US,en;q=0.9,es;q=0.8',
-            'Host': 'ing.ingdirect.es',
-            'Content-Type':'application/json; charset=UTF-8',
-            'Content-Length':'125',
-            'Connection': 'keep-alive',
-            'Origin': 'https://ing.ingdirect.es',
-            'Referer': 'https://ing.ingdirect.es/app-login/',
-            'Sec-Fetch-Mode': 'cors',
-            'Sec-Fetch-Site': 'same-origin',
-            'X-Requested-With': 'XMLHttpRequest',
-            }
+          'Accept':'*/*',
+          'User-Agent': 'Mozilla/5.0 (Linux; Android 4.2.1; en-us; Nexus 4 Build/JOP40D) AppleWebKit/535.19 (KHTML, like Gecko) Chrome/18.0.1025.166 Mobile Safari/535.19',
+          'Accept-Language': 'en-US,en;q=0.9,es;q=0.8',
+          'Host': 'ing.ingdirect.es',
+          'Content-Type':'application/json; charset=UTF-8',
+          'Origin': 'https://ing.ingdirect.es',
+          'Referer': 'https://ing.ingdirect.es/app-login/',
+          'Sec-Fetch-Mode': 'cors',
+          'Sec-Fetch-Site': 'same-origin',
+          'X-Requested-With': 'XMLHttpRequest',
+        }
```

Comment:
I changed the headers. Most of the things I don't think fixes nothing.

Accept => I saw the website only sending */*
Few of them I haven't seen the website sending them, I deleted them

Content-Length, I was worried about this one, cause it should be the length of the request in bytes as stated in RFC HTTP, I think it just does nothing because at HTTP level somehow will be overrided with the real length of the request. I just deleted just in case.